documentation/polymer-templates/tutorial-template-basic.asciidoc

```diff
@@ -47,100 +78,68 @@ class HelloWorld extends PolymerElement {
 
 customElements.define(HelloWorld.is, HelloWorld);
 ----
-
 * This is the JavaScript ES6 module that defines a Polymer template.
 * The `is()` function defines the name of the HTML tag that is used to reference this module. The tag name should contain at least one dash (`-`). For example, `hello-world` is a valid tag name, but `helloworld` is not.
-* The template contains two bindings: `\[[greeting]]` is a one-way binding and `{{userInput}}` is a two-way binding.
-* An event handler, `sayHello`, is triggered on a `helloButton` click.
-* No model is specified: it will be propagated automatically from the server side.
 * The imported dependencies are:
 ** `PolymerElement` (from the Polymer library): This is the required superclass of all Polymer templates.
 ** https://github.com/PolymerElements/paper-input[`paper-input`] Polymer component: You can use any component you like, for example the Vaadin `TextField`.
 ** `html` for inline DOM templating.
 
-[NOTE]
-This documentation covers Polymer 3 and npm in Vaadin version 14+ only. Earlier Vaadin versions support Polymer 2, HTML imports, and the Bower package manager. You can still use these components in Vaadin 14, if you work in the special *compatibility mode*. See the https://vaadin.com/docs/v13/flow/polymer-templates/tutorial-template-basic.html[Vaadin 13 documentation] for how to build client-side components with Polymer 2 and HTML imports.
-
-
-== Working on the Server Side
+== Creating the Java Template Class
 
-=== Creating the Java Template Class
+=== LitElement
 
-To use the client-side JavaScript template on the server side, you need to create an associated Java class that extends the `PolymerTemplate` class.
+To use the client-side JavaScript template on the server side, you need to create an associated Java class that extends the `LitTemplate` class.
```

Comment:
`client-side JavaScript` could be `LitElement` instead? We're talking specifically about LitElement here.

Reply:
Done.